test/types/range/elliot/anonymousRangeIter.expectedGenCode

```diff
@@ -13,33 +13,33 @@
 # loops to change.
 
 # for i in 1..2 do write(i); writeln();
-ic__F#_high_chpl = INT64(2);
-for (i_chpl = INT64(1); ((i_chpl <= _ic__F#_high_chpl)); i_chpl += INT64(1))
+ic__F#_high_chpl_anonymousRangeIter_chpl = INT64(2);
+for (i_chpl_anonymousRangeIter_chpl = INT64(1); ((i_chpl_anonymousRangeIter_chpl <= _ic__F#_high_chpl_anonymousRangeIter_chpl)); i_chpl_anonymousRangeIter_chpl += INT64(1))
```

Comment:
It looks like these names have already been uniquified in the old manner.  Is there a way to avoid the _chpl_<modName>_chpl double "chpl" output?